desktop/cmp/form/HoistField.js

```diff
@@ -176,4 +182,23 @@ export class HoistField extends Component {
         return ret;
     }
 
+    /**
+     * Override of the default implementation provided by HoistComponent so we can add the xh-field
+     * and xh-field-invalid classes (when appropriate)
+     */
+    getClassName(...extraClassNames) {
+        return classNames(this.baseClassName, this.props.className, ...extraClassNames, 'xh-field', this.invalid ? 'xh-field-invalid' : null);
+    }
+
+    /**
+     * @returns {boolean} whether the currently bound field's value is valid or not
```

Comment:
For clarity, suggest `true if the field's current value is invalid, false if valid or no validation model provided`.

Either that or we swap the sense around and make this `isValid()` which better matches the validationModel method.